_data/validator-libraries.yml

```diff
@@ -67,51 +74,62 @@
       license: "MIT"
     - name: hjsonschema
       url: https://github.com/seagreen/hjsonschema
-      notes: "- *supports draft 4*"
+      notes: 
+      draft: [4]
       license: MIT
 - name: Java
   implementations:
     - name: json-schema-validator
       url: https://github.com/java-json-tools/json-schema-validator
-      notes: "- *supports draft 4*"
+      notes: 
+      draft: [4]
       license: LGPLv3
     - name: json-schema (implementation based on the org.json API)
       url: https://github.com/everit-org/json-schema
-      notes: "- *supports draft 4, draft 6*"
+      notes: 
+      draft: [4, 6]
       license: Apache License 2.0
     - name: json-schema-validator
       url: https://github.com/networknt/json-schema-validator
-      notes: "- *supports draft 4*"
+      notes: 
+      draft: [4]
       license: Apache License 2.0
 - name: JavaScript
   implementations:
     - name: ajv
       url: https://github.com/epoberezkin/ajv
-      notes: "for Node.js and browsers - *supports draft 4, draft 6, [custom keywords](https://github.com/epoberezkin/ajv-keywords) and [$data reference](https://github.com/json-schema-org/json-schema-spec/issues/51)*"
+      notes: "for Node.js and browsers - *supports [custom keywords](https://github.com/epoberezkin/ajv-keywords) and [$data reference](https://github.com/json-schema-org/json-schema-spec/issues/51)*"
+      draft: [4, 5, 6]
```

Comment:
While @epoberezkin can correct me if I'm wrong, I'm pretty sure that AJV's "v5" was draft-04 plus the various keyword proposals that were floating around when the draft-04 authors left the project.  It's not entirely clear what it would even mean to say an implementation supported draft-05 as there was no formal way to indicate a schema used that draft, and the only validation change was adding clarifying "uri" and "uri-reference" as a format.